### PR TITLE
Add primecountpy

### PR DESCRIPTION
**Pull request by saraedum**
this is a new dependency of sagelib 9.5, see https://github.com/conda-forge/sagelib-feedstock/pull/129

### CODE REVIEW

File: recipes/primecountpy/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    - python
+  run:
+    - cysignals
+    - python
+
+test:
+  imports:
+    - primecountpy
+  commands:
+    - pip check
```
Comment (isuruf) [inline recipes/primecountpy/meta.yaml:37]:
```suggestion
```

**Comment by saraedum** [comment]
@isuruf, this seems to work.

File: recipes/primecountpy/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    - python
+  run:
+    - cysignals
+    - python
+
+test:
+  imports:
+    - primecountpy
+  requires:
+    - pip
```
Comment (isuruf) [inline recipes/primecountpy/meta.yaml:38]:
```suggestion
```

**Review by isuruf — APPROVED** [review]

File: recipes/primecountpy/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+  build:
+    - python                                 # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
+    - cython                                 # [build_platform != target_platform]
+    - {{ compiler('cxx') }}
+  host:
+    - cysignals
+    - cython
+    - pip
+    - primecount 7.2
```
Comment (hmaarrfk) [inline recipes/primecountpy/meta.yaml:28]: You should add this pinning to:
https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/master/recipe/conda_build_config.yaml

**Pull request merged**